apps/frontend/src/modules/auth/AuthProvider.tsx

```diff
@@ -352,21 +368,22 @@ export const AuthProvider = ({ children, apollo }: AuthProviderProps) => {
 
     useEffect(() => {
         const hydrate = async () => {
-            if (!!refreshToken && !hydrated) {
+            if (!!refreshToken) {
                 const { data } = await getUser();
-                if (data?.viewer) {
-                    const organization = data.viewer.organizations?.edges.find(
-                        ({ node }) => node.id === user.organization?.id,
-                    )?.node;
-                    const project = organization?.projects?.edges.find(
-                        ({ node }) => node.id === user.project?.id,
-                    )?.node;
-                    updateUserCache({
-                        ...data.viewer,
-                        organization: convertToAuthOrganization(organization),
-                        project,
-                    });
-                }
+                // if (data?.viewer) {
+                //     const organization = data.viewer.organizations?.edges.find(
```

Comment:
@afolasope Why did you remove this? If it's no longer needed, please remove them instead of commenting them out.

Reply:
I have attended to this.